src/content/1.7/faq/stock.md

```diff
@@ -0,0 +1,77 @@
+
+{{% notice note %}}
+You will probably notice that usually ```physical_quantity = available_quantity + reserved_quantity;```
+{{% /notice %}}
+
+**Q:** When is the StockAvailable object/table updated?
+
+**A:** That's a broad question but basically for any action that should logically have an impact on the stock. When a product quantity is modified in BO, when an order is created, when the order is delivered, ... However here is a little chronology of stock quantities during a full checkout process:
+
+| Scenario Step | quantity | physical_quantity | reserved_quantity |   |
+| ------------- | -------- | ----------------- | ----------------- | - |
+| `Great Product` is created in the BO with a quantity of 50 | `50` | `50` | `0` | |
+| A user decides to add 3 `Great Product` into his cart      | `50` | `50` | `0` | For now the product is only in a cart, nothing is reserved anyone can still buy one of the 50 remaining products. |
+| The user goes through the whole checkout process and chooses to pay by check. | `50` | `47` | `3` | The order has been created, but the payment is not accepted yet. |
+| A few days later, the check is received, and the merchant changes the Order status to `Payment accepted` | `50` | `47` | `3` | The product is now paid, but it still has not left your shelves. |
+| The package is prepared and delivered to a Carrier, the order status is switched to `Delivered` or `Shipped` | `47` | `47` | `0` | The product is no longer reserved, it has actually been sent to the customer. So reserved and physical quantities can be decreased. |
+
+**Q:** So where can I see and modify my Stock?
+
+**A:** Although you can change the quantity directly in your product page we strongly recommend using the `Catalog > Stock` section in your BO because it is based on `StockMovement` which is more stable than direct edition.
+
+## StockMovement
+
+**Q:** But what is a `StockMovement`? What is the difference with direct edition?
+
+**A:** When you edit the quantity of a Product you are going to edit the StockAvailable::quantity field directly in your database. When you use a modification with a StockMovement you apply a delta to your existing stock.
+
+**Q:** And what is the advantage of this delta edition?
+
+**A:** When you edit the quantity the total value is sent and saved in database, but you cannot be sure that there was no modification between the moment you open your edition page, and the moment you save the new expected value.  Imagine this scenario:
+
+- You have a 50 `Great Product` in your stock.
+- It is your best seller, so today you are expecting a delivery to increase your stock.
+- You get 50 new products and want to udpate the stock from the Product page.
+- You open the page indicating that you have 50 products left in your stock.
+- While you are checking this stock, since it is your best seller, three products have been sold and delivered so your stock is now actually 47.
```

Comment:
```suggestion
- While you are checking this stock, since it is your best seller, three items have been sold and shipped, so your stock is now actually 47, not 50.
```